SETUP.md

```diff
@@ -109,3 +111,32 @@ For the `devise` and `administrate` files, there might be available translations
 If you want to change the default language for Terrastories, set the language on line 21 in `rails/config/application.rb`. To set it to Papiamentu, change this line to `config.i18n.default_locale = :pap`
 
 Once you are done, the language should be available the next time you start Terrastories.
+
+## Testing
+
+The terrastories application is currently using rspec for testing, to run existing tests make sure you are within
+the docker container (run `$ docker exec terrastories_web_1 /bin/bash`) then run
+
+```
+$ docker-compose run web scripts/wait-for-it.sh db:5432 -- "rspec spec"
+```
+
```

Comment:
I had trouble running this when I was on a new branch and needed to install gems. The way that I was able to run it was to do:
`docker exec -it terrastories_web_1 /bin/bas`
which put me into the rails container command line and then directly run `rspec spec` The other command was fine when I didn't need any new gems installed. What do you think of adding this as another way of running the tests in the container?